Website/views/pages/admin.ejs

```diff
@@ -21,19 +21,32 @@
                 <tr>
                     <th>Account ID</th>
                     <th>User ID</th>
+                    <th>Username</th>
                     <th>Routing Number</th>
                     <th>Account Number</th>
                     <th>Balance</th>
                 </tr>
             </thead>
             <% for(var i=0; i < data.length; i++) { %>
-                <tr>
-                    <td><%= data[i].account_id %></td>
-                    <td><%= data[i].user_id %></td>
-                    <td><%= data[i].routing_number %></td>
-                    <td><%= data[i].account_number %></td>
-                    <td><%= data[i].balance %></td>
-                </tr>
+                <% if(secure) { %>
```

Comment:
What's the difference here? Am I overlooking something or are the two branches the same?

Reply:
EJS has built in XSS measures. The "<%=" syntax encodes anything after it, preventing a malicious script from executing. The "<%-" syntax does not encode anything, so it's up to the browser to render it, which will execute scripts.

Reply:
Didn't notice the "=" vs the "-" difference, oops. Thanks!